### PR TITLE
(SLV-788) New and updated dashboards

### PR DESCRIPTION
**Pull request by RandellP**
added dashboards for system and process data that was recently added to Puppet metrics collector
added some new dashboards for ace and bolt data which was recently added to puppet metrics collector
Added some new dashboards for deeper dives into some of the jvms and puma services
Added a new orchestration services dashboard for ace, bolt and orch performance data.

### CODE REVIEW

File: manifests/dashboards/puppet_metrics.pp [hunk 1]
```diff
@@ -25,8 +35,34 @@
...
+    'Archive Orchestration Services':
+      content => file('puppet_metrics_dashboard/Orchestration_Services.json'),
+    ;
+    'Archive Process/System Stats':
+      content => file('puppet_metrics_dashboard/Process_System_Stats.json'),
+    ;
     'Archive File Sync Metrics':
       content => file('puppet_metrics_dashboard/Archive_File_Sync.json'),
     ;
+    'Archive Ace Puma Performance ':
```
Comment (jarretlavallee) [inline manifests/dashboards/puppet_metrics.pp]: There is a trailing space in this dashboard name

**Comment by tkishel** [comment]
I think we need to "see" these :)

**Comment by RandellP** [comment]
http://10.234.1.30:3000/dashboards   But don't mess anything up... I need to record a demo using that. :)

**Comment by johnduarte** [comment]
Demo completed. Go nuts @tkishel !

**Comment by jarretlavallee** [comment]
These are looking great. The linking on the new dashboard is a bit odd. Some are linked and others are not. Should all of the deeper dives be linked?

**Comment by RandellP** [comment]
@jarretlavallee I tried to link the ones that were most likely to be the next step.  Like if you were in orch services, you would go to the ace bolt or orch deep dive, not likely to go to the puppetserver deep dive from there.

**Comment by jarretlavallee** [comment]
@Sharpie Thoughts on these?

**Comment by RandellP** [comment]
@jarretlavallee   I updated this one GC graph... http://10.234.1.30:3000/d/kI83OvUWk/archive-puppetserver-jvm-performance  But I couldn't figure out how to make a trend line.  Googling implied I couldn't without munging the data in the db...

On the display sort... I can do that.  Though I have no data to test, but I think that would mean that it would sort uniquely for each graph.  Which would mean the order of servers would be different for each graph.  Is that okay?

On the file sync stuff.  I don't have any data from my runs, so it is hard for me to add those.  Maybe should be a separate PR.

**Comment by RandellP** [comment]
@tkishel On the orchestrator thread pool.  Ace is already there... I had it use the right side axes so that it could have a different range and be more visible.  But maybe that made it hard to find?    And what do you mean by verify?
I moved file sync to deeper dives and pushed that commit.

**Comment by jarretlavallee** [comment]
@RandellP That looks good. I made a quick right axis on the GC for the time. How does it look now? http://10.234.1.30:3000/d/kI83OvUWk/archive-puppetserver-jvm-performance?panelId=14&orgId=1&tab=general

The other suggestions are fine to leave off for now. We can revisit if the need comes up with a follow up PR.

**Comment by RandellP** [comment]
@jarretlavallee I just pushed up the changes for the gc-stats... I still need to test, but somehow my bolt installation on my laptop got updated and my plan that does the update doesn't work... so I will have to fix that later and test.   Need Tom's answers anyway before we merge... :)

**Pull request merged**